R/statistic-histogram.R

```diff
@@ -2,6 +2,32 @@
 #' 
 #' Determine accuracy of histogram release, given epsilon and delta, for the differentially 
 #' private histogram release.
+#' 
+#' In differential privacy, "accuracy" is defined as the threshold value above which a given value 
+#' is "significantly different" from the expected value. Mathematically, this is written as:
+#' \deqn{\alpha = Pr[Y > a]}
+#' Where \eqn{\alpha} is the statistical significance level, \eqn{a} is the accuracy, and \eqn{Y} is a random 
```

Comment:
Give definition of Y (since it's technically the difference between the noisy output and true output; i.e. has got that absolute value in it, so is pulled from folded Laplace not normal Laplace).